client/src/pages/login.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Button, Checkbox, Form } from 'semantic-ui-react'
+
```

Comment:
Could you change names of components from generic to the proper names we are going to use

Reply:
Does this still need to be done? I can finish this ticket if needed.